Wiki/FAQ.md

```diff
@@ -17,9 +17,6 @@ You will see a lot of references to the data folder in this FAQ here are the def
 Almost all of your computing power can be directed at beneficial scientific and medical
 research while crunching Gridcoin.
 
-See the list of [Advantages &
-Features](Advantages_&_Features "wikilink").

```

Comment:
I am working on creating a new version of this page, so we can still keep it in even if the link is dead for a little bit

Reply:
sure thing